services/repository-managers/src/test/resources/ses/message_no_mail.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Type" : "Notification",
+  "MessageId" : "edb5b81e-e48a-5fa2-9cb0-ee16f03d03bb",
+  "TopicArn" : "arn:aws:sns:us-east-1:3222221231312:Some-Topic",
+  "Message" : "{\"notificationType\":\"Bounce\",\"bounce\":{\"bounceType\":\"Permanent\",\"bounceSubType\":\"General\",\"bouncedRecipients\":[{\"status\":\"5.0.0\",\"action\":\"failed\",\"diagnosticCode\":\"smtp; 550 user unknown\",\"emailAddress\":\"recipient1@example.com\"},{\"status\":\"4.0.0\",\"action\":\"delayed\",\"emailAddress\":\"recipient2@example.com\"}],\"timestamp\":\"2012-05-25T14:59:38.605Z\",\"feedbackId\":\"000001378603176d-5a4b5ad9-6f30-4198-a8c3-b1eb0c270a1d-000000\"}}",

```

Comment:
support raw message (sent to queue without going through topic), then tests like this can be simple formatted json.